### PR TITLE
Added totals

### PR DESCRIPTION
**Pull request by a-nahaitsev**

### CODE REVIEW

File: renderer/components/Totals.tsx
Comment (d-mashchenko) [inline renderer/components/Totals.tsx:1]: Some clients want to see spent time only hours, i mean 15m => 0.25. Would be nice approve this.

Comment (d-mashchenko) [reply renderer/components/Totals.tsx:1]: As idea: don't show arrow if refistration does not have activity.

Comment (d-mashchenko) [reply renderer/components/Totals.tsx:1]: Also you can chagne title of copy icon. If we copy all descriptions for registration and if we copy description only for separated acitvity.

Comment (a-nahaitsev) [reply renderer/components/Totals.tsx:1]: @d-mashchenko 
2 last comments - done
The first one - i left as was before (we discussed it on the meeting)

**Review by o-mehlii — APPROVED** [review]

**Review by d-mashchenko — APPROVED** [review]

**Pull request merged**